docs/modules/ROOT/pages/list-of-metrics.adoc

```diff
@@ -105,6 +105,7 @@ in milliseconds).
 
 |userCancelled
 |Details on whether the user canceled the job. Potential values: 1 - canceled by the user, 0 - otherwise.
+Expose via xref:maintain-cluster:monitoring.adoc#via-job-api[Job API] only.
```

Comment:
I'd prefer "using" over "via", but I note that the heading of the linked topic is "Via Job API", so I won't insist ;-)